posydon/binary_evol/MESA/step_mesa.py

```diff
@@ -1325,36 +1355,125 @@ def __call__(self, binary):
         if (state_2 in ["WD", "NS", "BH"]
```

Comment:
Could you add a TODO to import the list of compact objects from flow_chart.py as well (please do it for all cases with a static list `["WD", "NS", "BH"]`.

Reply:
done